tokio/tests/time_interval.rs

```diff
@@ -0,0 +1,45 @@
+#![warn(rust_2018_idioms)]
```

Comment:
This is renamed from `timer_interval.rs` and updated to the API changes. I guess those were too significant and it registered as a new file.